features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -201,7 +205,16 @@ internal class RumViewScopeTest {
     private lateinit var mockNetworkSettledMetricResolver: NetworkSettledMetricResolver
 
     @Mock
-    private lateinit var mockInteractionToNextViewMetricResolver: InteractionToNextViewMetricResolver
+    private lateinit var mockInteractionNextViewMetricResolver: InteractionToNextViewMetricResolver
```

Comment:
is it expected rename? Should't it still have `To` in the name as in the interface name?

Reply:
looks like auto-renamed by AS